specs-go/config.go

```diff
@@ -788,3 +818,43 @@ type ZOSDevice struct {
 	// Gid of the device.
 	GID *uint32 `json:"gid,omitempty"`
 }
+
+// LinuxSchedulerPolicy represents different scheduling policies used with the Linux Scheduler
+type LinuxSchedulerPolicy string
+
+const (
+	// SCHED_OTHER is the default scheduling policy
+	SCHED_OTHER LinuxSchedulerPolicy = "SCHED_OTHER"
```

Comment:
```suggestion
	SchedOther LinuxSchedulerPolicy = "SCHED_OTHER"
```
To follow the current coding style:
https://github.com/opencontainers/runtime-spec/blob/main/specs-go/config.go#L677

Reply:
thanks, changed to camel case